tests/test_hubspot_bookmarks.py

```diff
@@ -116,11 +123,13 @@ def test_run(self):
             )
             self.expected_records['contacts_by_company'] += record
 
-
         # Update 1 record from the test seutp for each stream that has an update endpoint
+        # Update the last record that was initially created, instead of first record
+        # - Workaround for https://jira.talendforge.org/browse/TDL-24615
         for stream in expected_streams - STREAMS_WITHOUT_UPDATES:
             primary_key = list(self.expected_primary_keys()[stream])[0]
-            record_id = self.expected_records[stream][0][primary_key]
+            index_to_update = len(self.expected_records[stream])-1
```

Comment:
This workaround will affect all streams and not just the contact lists.  And the last record created will be synced regardless of it is updated or not so this isn't testing this for other streams. I think a better workaround would be to add this stream to STREAMS_WITHOUT_UPDATES so this loop just doesn't run for contact lists until it works.

Reply:
Changed it to be the first record as before.